plugins/action/jcliff.py

```diff
@@ -70,8 +70,12 @@ def _lookup_subsys_template(self, subsys_name):
 
     def _build_and_deploy_jcliff_rule_files(self, tmp_remote_src):
         display.vvvv(u"Build and deploy jcliff rule tmpfile: %s" % tmp_remote_src)
-        display.vvvv(u"Subsystems: %s" % self._task.args['subsystems'])
-        subsystems = self._task.args['subsystems']
+
+        if 'subsystems' in self._task.args:
+            subsystems = self._task.args['subsystems']
+        if 'components' in self._task.args:
+            subsystems = self._task.args['components']
+
         if subsystems is not None:
```

Comment:
Should change variable to components

Reply:
Yes, and this is why my previous change already. But I would rather do that in a separate commit/PR.